AzureMonitorAgent/ama_tst/modules/main.py

```diff
@@ -41,15 +44,19 @@ def run_troubleshooter():
     while (run_again):
         print("================================================================================\n"\
             # TODO: come up with scenarios
-              "(NOTE: troubleshooting scenarios are coming, currently only log collection is available)\n"\
+              "1: Installation failures. \n"\
+              "2: Onboarding failures. \n"\
```

Comment:
Nitpick - would remove the "Onboarding failures" line until the onboarding section has been completed, may be confusing to customers / CSS